test/ejabberd_tests/tests/ejabberdctl_SUITE.erl

```diff
@@ -527,18 +544,21 @@ process_rosteritems_list_advanced1(Config) ->
         {KateName, Domain, _} = get_user_data(kate, Config),
         ContactMike = string:to_lower(binary_to_list(escalus_client:short_jid(Mike))),
         ContactKate= string:to_lower(binary_to_list(escalus_client:short_jid(Kate))),
-        ContactsRegexp = ContactMike ++ ":" ++ string:substr(binary_to_list(KateName), 1, 2) ++ ".*@.*",
+        ContactsRegexp = ContactMike ++ ":" ++
+                      string:substr(binary_to_list(KateName), 1, 2) ++ ".*@.*",
```

Comment:
Indentation, `string` should start in the same column as `ContactMike` in the line above.